go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/go-logr/zapr v0.4.0
 	github.com/google/go-cmp v0.5.6
 	github.com/kevinburke/go-bindata v3.11.0+incompatible
-	github.com/openshift/api v0.0.0-20210722202039-91e26cfb36fc
+	github.com/openshift/api v0.0.0-20210924154557-a4f696157341
```

Comment:
You don't have to take the latest commit, just take the latest one that you need for this PR.  Isn't it https://github.com/openshift/api/commit/c9d6fcc62893a426618b74c3ce415e5ae045f7e6 ?

Reply:
Yes, I should take this one.

But the version of k8s.io/api has already bumped to 0.22.1 before this commit in openshift/api@6448676 , with an update of client-go and kube-openapi, which implemented a breaking change, causes the failure of compilation.

https://github.com/openshift/api/blob/c9d6fcc62893a426618b74c3ce415e5ae045f7e6/go.mod#L12

Maybe we should wait for the dependency issue to be resolved and then continue?